_examples/digitalidentity/login.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html class="yoti-html" lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <title>Yoti client example</title>
+    <link rel="stylesheet" type="text/css" href="/static/index.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet"/>
+</head>
+<body class="yoti-body">
+    <main>
+        <section class="yoti-top-section">
+          <div class="yoti-logo-section">
+            <img class="yoti-logo-image" src="/static/assets/logo.png" srcset="/static/assets/logo@2x.png 2x" alt="Yoti"/>
+          </div>
+
+          <h2 th:text="${message}">Digital Identity Share Example page</h2>
+
+          <p> SdkId: <strong th:text={{.yotiClientSdkID}}>{{.yotiClientSdkID}}</strong></p>
+
+          <div class="yoti-sdk-integration-section">
+            <div id="webshare-target"></div>
+          </div>
+
+        </section>
+
+        <section class="yoti-sponsor-app-section">
+          <h3 class="yoti-sponsor-app-header">The Yoti app is free to download and use:</h3>
+
+          <div class="yoti-store-buttons-section">
+            <a href="https://itunes.apple.com/us/app/yoti/id983980808?ls=1&amp;mt=8" class="yoti-app-button-link">
+              <img src="/static/assets/app-store-badge.png" srcset="/static/assets/app-store-badge@2x.png 2x"
+                   alt="Download on the App Store"/>
+            </a>
+
+            <a href="https://play.google.com/store/apps/details?id=com.yoti.mobile.android.live"
+               class="yoti-app-button-link">
+              <img src="/static/assets/google-play-badge.png" srcset="/static/assets/google-play-badge@2x.png 2x"
+                   alt="get it on Google Play"/>
+            </a>
+          </div>
+        </section>
+      </main>
+<script>
+    async function onSessionIdResolver() {
+        console.log('sessioning')
+        const response = await fetch('https://localhost:8080/v2/generateShare', {method: 'GET'})
+        const data = await response.json()
+        console.log('data: %s', data.id)
+        return data.id
+    }
+
+    async function completionHandler(receivedReceiptId) {
+        console.log('completion handler:', receivedReceiptId)
+        const response = await fetch('/v2/receiptInfo?ReceiptID=' + encodeURIComponent(receivedReceiptId) , {method: 'GET'})
```

Comment:
don't forget to update the frontend calls to the API since we updated the path from camelcase